src/mmw/js/src/app.js

```diff
@@ -64,6 +65,34 @@ var App = new Marionette.Application({
 
     hideLoginModal: function() {
         $('#login').modal('hide');
+    },
+
+    userLogout: function() {
```

Comment:
App has a reference to user already so I think you'd probably want to do something like call user.logout() since there may be other areas of the application later that also cause a logout to occur.